airflow/migrations/versions/0063_3_0_0_use_ti_id_as_fk_to_taskreschedule.py

```diff
@@ -0,0 +1,135 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Use ti_id as FK to TaskReschedule.
+
+Revision ID: d469d27e2a64
+Revises: 16f7f5ee874e
+Create Date: 2025-03-06 16:04:49.106274
+
+"""
+
+from __future__ import annotations
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+from airflow.migrations.db_types import StringID
+
+# revision identifiers, used by Alembic.
+revision = "d469d27e2a64"
+down_revision = "16f7f5ee874e"
+branch_labels = None
+depends_on = None
+airflow_version = "3.0.0"
+
+
+def upgrade():
+    """Apply Use ti_id as FK to TaskReschedule."""
+    with op.batch_alter_table("task_reschedule", schema=None) as batch_op:
+        batch_op.add_column(
+            sa.Column(
+                "ti_id", sa.String(length=36).with_variant(postgresql.UUID(), "postgresql"), nullable=False
```

Comment:
Shouldn't we populate this column from the current pk columns?

Reply:
Yeah, my bad